RecoParticleFlow/PFClusterProducer/plugins/alpaka/PFClusterSoAProducer.cc

```diff
@@ -75,8 +102,11 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE {
     const edm::EDGetTokenT<reco::PFRecHitHostCollection> inputPFRecHitSoA_Token_;
     const device::EDPutToken<reco::PFClusterDeviceCollection> outputPFClusterSoA_Token_;
     const device::EDPutToken<reco::PFRecHitFractionDeviceCollection> outputPFRHFractionSoA_Token_;
+    cms::alpakatools::host_buffer<uint32_t> num_rhf_;
```

Comment:
This is really nitpicking, feel free to ignore me:
```suggestion
    cms::alpakatools::host_buffer<uint32_t> numRHF_;
```
to follow the way the other data members are named.

Reply:
This naming was also changed